### PR TITLE
{devel}[GCCcore/13.2.0] libabigail v2.5

### PR DESCRIPTION
**Pull request by ocaisa**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3

**Comment by ocaisa** [comment]
This could be a useful tool if you want to do MPI library replacement like we allow for in EESSI. It allows you to compare the ABI of shared object libraries and check their compatibility.

File: easybuild/easyconfigs/l/libabigail/libabigail-2.5-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+configopts = '--enable-manual=no --enable-apidoc=no --with-sysroot=%(sysroot)s'
+
+sanity_check_paths = {
+    'files': [
+        'bin/abicompat', 'bin/abidiff',
+        'lib/libabigail.a', 'lib/libabigail.%s' % SHLIB_EXT,
+    ],
+    'dirs': ['include'],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/l/libabigail/libabigail-2.5-GCCcore-13.2.0.eb:47]: @ocaisa is it possible to add `sanity_check_commands` for `abicompat` and `abidiff` here?

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @ocaisa!

**Pull request merged**